cpp/src/arrow/compute/kernels/scalar_string_test.cc

```diff
@@ -1041,6 +1037,73 @@ TYPED_TEST(TestStringKernels, Utf8Title) {
       R"([null, "", "B", "Aaaz;Zææ&", "Ɑɽɽow", "Ii", "Ⱥ.Ⱥ.Ⱥ..Ⱥ", "Hello, World!", "Foo   Bar;Héhé0Zop", "!%$^.,;"])");
 }
 
+TYPED_TEST(TestStringKernels, StringRepeatWithScalarRepeat) {
```

Comment:
Is there a place where passing a scalar for the strings argument is tested? Is it implicit in `CheckVarArgs`?

Reply:
Yes, it is implicit in `CheckVarArgs`. `CheckVarArgs` invokes [`CheckScalar` which internally calls function for each scalar input](https://github.com/apache/arrow/blob/master/cpp/src/arrow/compute/kernels/test_util.cc#L127).